### DISCUSSION TITLE
还有人吗qwq

### DISCUSSION THREAD

**Discussion by cuijiaqi0406** [share121/inter-knot#5612] [公告]
<img src="https://bbs-static.miyoushe.com/static/2024/03/21/a63b1fdb4c2a4562c7d291b230978f0a_944582820103199731.png" alt="丽都漫步-莱卡恩" width="75" >话题只剩投票，但是《暂不支持》

**Comment by scrooge-0** [comment]
还是热度过喽，我也是才想起来还有这个功能，毕竟不是主要社交软件（哭笑不得）

**Comment by xinggif** [comment]
还真是

**Comment by JT9364** [comment]
成为小圈子（笑，但是为什么还是有广告狗啊（
